ehitchhiking-ui/src/app/shared/api-services/black-list-api.service.ts

```diff
@@ -0,0 +1,54 @@
+import {User} from '@shared/models/user';
+import {URL_REGISTRY} from '@shared/constants/urlRegistry';
+import {ApiService} from '@shared/services/api.service';
+import {Car} from '@shared/models/car';
+import {map} from 'rxjs/operators';
+
+export class BlackListApiService {
+	constructor(private apiService: ApiService) {}
+
+	makeArrayPassengers(data) {
+		let blacklistPass = [];
```

Comment:
There is no reason to create a temp variable. `map` returns and array

Reply:
removed